src/Foundation/NSRange.cs

```diff
@@ -36,6 +36,21 @@ public NSRange (nint start, nint len)
 			Length = len;
 		}
 
+		public override int GetHashCode()
+		{
+			return HashCode.Combine(Location, Length);
+		}
+		
+		public override bool Equals(object obj)
+		{
+			return obj is NSRange other && Equals(other);
+		}
+		
+		public override bool Equals(NSRange other)
```

Comment:
there should not be any need to `override` this member

Reply:
Thank you, this was muscle memory at play.